src/pages/simple-meeting/App.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useRef, useEffect, useCallback } from 'react';
+import styles from './submitForm.module.css';
+import TokenButton from '../../navigation/TokenButton/TokenButton';
+import MenuButton from '../../navigation/MenuButton/MenuButton';
+import * as Callbridge from '@iotum/callbridge-js';
+import { useSelector } from 'react-redux';
+
+const App = () => {
+  const [accessCode, setAccessCode] = useState('');
+  const container = useRef(null);
+  const widget = useRef(null);
+
+  const credentials = useSelector(state => state.credentials);
+
+  const handleAccessCode = (event) => {
+    setAccessCode(event.target.value);
+  }
+
+  const renderWidget = useCallback(() => {
+    console.log("render meeting widget");
+
+    if (widget.current) {
+      widget.current.unload();
+    }
+
+    widget.current = new Callbridge.Meeting(
+      {
+        domain: credentials.domain, 
+        sso: {
+          token: credentials.token,
+          hostId: credentials.hostId
+        },
+        container: container.current,
+      },
+      accessCode, 
+      {
+        skipJoin: true
+      }
+    );
+  }, [credentials, accessCode]); // Dependencies: credentials and accessCode
+
+  useEffect(() => {
```

Comment:
this is wrong. using `renderWidget` as dependency means this gets loaded whenever input.onChange fires.

this sample should not use `useEffect` and the loading should be based on button.onClick only